### PR TITLE
fixing button for mdc0.21.0

### PR DESCRIPTION
**Pull request by prateekbh**
fixes 1 and 2 from #263

### CODE REVIEW

File: Button/Button.jsx [hunk 1]
```diff
@@ -1,28 +1,19 @@
 import { h } from "preact";
 import MaterialComponent from "../MaterialComponent";
-
+import Icon from "../Icon/";
 /**
```
Comment (filoozom) [inline Button/Button.jsx]: Tiny detail but maybe add a new line after this 😛

**Comment by prateekbh** [comment]
@filoozom done!

**Comment by prateekbh** [comment]
@filoozom if i updated docs now and merged it to master, it'll search for the new version before its published and might break... isnt it?

**Comment by filoozom** [comment]
Oh, right, didn't even think about that... We might need a deploy script for ```npm``` or push the commit after the release, but that actually doesn't make it any easier to update docs at the same time 😞.

**Comment by prateekbh** [comment]
@filoozom if we do that, the problem will be that there might be a point where it does not point to the published version but the the one in WIP. this might confuse people.

**Pull request merged**